host/4/mariner/dotnet/dotnet-isolated/dotnet7-isolated.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.21.1
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+RUN yum install -y dnf && \
```

Comment:
I don't believe that these statements are doing anything because of where they are.  They install some toolin gto the host image. But we don't copy them over or anything so they are just wasted space rn

Reply:
As a result of this you have to do the line 25 dnf install again and our runtime images don't have curl wget or unzip.  However, those may not be necessary at all since we don't have to handle the ext bundles for this image

Reply:
True, I'll add it for the last image

Reply:
Wait, doesn't this mean that it's the same case for our current dotnet-isolated images? I was basically trying to replicate our current dotnet7-isolated image with a mariner base